resources/assets/components/utilities/SiteFooter/SiteFooter.js

```diff
@@ -96,74 +108,189 @@ const SiteFooter = () => {
+              <a
+                href="/us/about/our-financials"
+                onClick={event =>
+                  handleFooterTracking('our_financials', event.target.href)
+                }
+              >
+                Our Financials
+              </a>
             </li>
             <li>
-              <a href="/us/about/our-press">Press</a>
+              <a
+                href="/us/about/our-press"
+                onClick={event =>
+                  handleFooterTracking('our_press', event.target.href)
+                }
+              >
+                Press
+              </a>
             </li>
             <li>
-              <a href="https://lets.dosomething.org/">Articles</a>
+              <a
+                href="https://lets.dosomething.org/"
+                onClick={event =>
+                  handleFooterTracking('articles', event.target.href)
+                }
+              >
+                Articles
+              </a>
             </li>
             <li>
-              <a href="/us/about/contact-us">Contact Us</a>
+              <a
+                href="/us/about/contact-us"
+                onClick={event =>
+                  handleFooterTracking('contact_us', event.target.href)
+                }
+              >
+                Contact Us
+              </a>
             </li>
           </ul>
         </div>
         <div className="footer__column -links">
           <h4>Our Friends</h4>
           <ul>
             <li>
-              <a href="http://dosomethingstrategic.org/">
+              <a
+                href="http://dosomethingstrategic.org/"
+                onClick={event =>
+                  handleFooterTracking('our_friends', event.target.href)
+                }
+              >
                 DoSomethingStrategic.org
               </a>
             </li>
             <li>
-              <a href="/us/about/our-partners">Partners</a>
+              <a
+                href="/us/about/our-partners"
+                onClick={event =>
+                  handleFooterTracking('our_partners', event.target.href)
+                }
+              >
+                Partners
+              </a>
             </li>
             <li>
-              <a href="/us/about/hotline-list">Crisis Hotlines</a>
+              <a
+                href="/us/about/hotline-list"
+                onClick={event =>
+                  handleFooterTracking('crisis_hotlines', event.target.href)
+                }
+              >
+                Crisis Hotlines
+              </a>
             </li>
           </ul>
         </div>
         <div className="footer__column -links">
           <h4>Get Involved</h4>
           <ul>
             <li>
-              <a href="/us/articles/clubs-notify-me">DoSomething Clubs</a>
+              <a
+                href="/us/articles/clubs-notify-me"
+                onClick={event =>
+                  handleFooterTracking('doSomething_clubs', event.target.href)
```

Comment:
We need to keep the `dosomething` downcase otherwise the `do` and `something` will be separated by an underscore 😱 
```suggestion
                  handleFooterTracking('dosomething_clubs', event.target.href)
```